app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -163,7 +158,21 @@ define(function (require, exports, module) {
     },
 
     deleteDisplayedAccountProfileImage (account) {
-      return account.deleteAvatar(this._displayedProfileImage.get('id'))
+      return p()
+        .then(() => {
+          if ( ! account.get('profileImageId') ) {
```

Comment:
review nit: no space before ` !` or here `account.get('profileImageId') )` .

Remove the space before the last parenthesis.  

@pdehaan is there an ESlint rule that can help us track these?